org-jira.el

```diff
@@ -520,6 +520,7 @@ See `org-default-priority' for more info."
     (define-key org-jira-map (kbd "C-c iw") 'org-jira-progress-issue)
     (define-key org-jira-map (kbd "C-c in") 'org-jira-progress-issue-next)
     (define-key org-jira-map (kbd "C-c ia") 'org-jira-assign-issue)
+    ;(define-key org-jira-map (kbd "C-c io") 'org-jira-report-issue)
```

Comment:
I'm closely following the convention, (from `org-jira-assign-issue`), here but do have some concern that a function named `org-jira-report-issue` could be confused with `org-jira-create-issue`.

Listening for feedback on this one.